modules/virt-understanding-smart-cloning.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-understanding-smart-cloning_{context}"]
+= Understanding smart cloning
```

Comment:
There is still an instance of "smart cloning" (no hyphen) here.

Reply:
Duh... Sorry. Fixed.